services/brig/src/Brig/API/User.hs

```diff
@@ -538,13 +538,14 @@ initAccountFeatureConfig uid = do
 createUserInviteViaScim ::
   Members
     '[ BlacklistStore,
-       UserPendingActivationStore p
+       UserPendingActivationStore p,
+       GalleyProvider
      ]
     r =>
   UserId ->
   NewUserScimInvitation ->
   ExceptT Error.Error (AppT r) UserAccount
-createUserInviteViaScim uid (NewUserScimInvitation tid loc name rawEmail) = do
+createUserInviteViaScim uid (NewUserScimInvitation tid loc name rawEmail _) = do
```

Comment:
given you've tested this well, you probably don't want to write the role to cassandra here, but...  why?  i'm confused.  :)

Reply:
The role is stored in C* just before this function is called.
In this function we create an entry in the pending activation store where we do not need the role, but since it's part of the type it needs to be included in the deconstructor.